README.md

```diff
@@ -33,6 +33,7 @@ _These assume you have `egghead-systems` running locally_
 - **Package management:** Yarn
 - **Scripts:** Yarn
 - **JS Flavor:** ES2016+ _(default `create-react-app`)_
+- **JS Utils:** Lodash
```

Comment:
Like...would you have to manually kill the observable each time after you use an operator? Not sure how that works...